### PR TITLE
Update Dockerfile to use the ADD command

### PR DESCRIPTION
**Pull request by TRex22**
# What does this PR do?
Im not sure why but the first two wget commands kept failing for me. Using `ADD` resolved the issue.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,6 +4,10 @@ ARG NOVNC_VERSION="1.4.0"
 ARG OPENWRT_VERSION="23.05.3"
 ARG VERSION_ARG="0.1"
 
+RUN mkdir -p /tmp/noVNC-${NOVNC_VERSION}
+ADD https://github.com/novnc/noVNC/archive/refs/tags/v${NOVNC_VERSION}.tar.gz /tmp/novnc.tar.gz
+ADD https://github.com/bugy/script-server/releases/download/1.18.0/script-server.zip /tmp/noVNC-${NOVNC_VERSION}/script-server.zip
```
Comment (AlbrechtL) [inline Dockerfile:9]: It doesn't make sense to put script-server.zip into the noVNC folder. Can you change that, please?

In the original code it doesn't make sense either.

**Review by AlbrechtL — CHANGES_REQUESTED** [review]
Thanks for the improvements. I also don't know why your `wget` failed. Maybe `wget` is running inside the alpine and not at the host - but this is just guessing.

I added a comment inside your change.

**Comment by AlbrechtL** [comment]
Closed because of no activity.

**Comment by TRex22** [comment]
Sorry @AlbrechtL been a busy year and I lost track of stuff...
Im busy trying to add a few things here and will open some PRs and do your feedback when Im ready to

**Pull request closed without merging**